try:
    print('try')
except ZeroDivisionError:
    print('except 1')
finally:
    print('finally 1')
# finally:
#     print('finally 2')


# This is Invalid
# We can't take multiple finally

# try-except-else-finally
# try : take only 1 time
# except : take N number of times
# else : take only 1 time
# finally : take only 1 time



